src/babel/transformation/transformers/es7/function-bind.js

```diff
@@ -0,0 +1,37 @@
+// https://github.com/zenparsing/es-function-bind
+
+import * as t from "../../../types";
+
+export var metadata = {
+  optional: true,
+  stage: 0
+};
+
+var CALL = t.identifier("call");
```

Comment:
The AST is currently considered to be highly mutable so this isn't really safe to do. ie. properties could be added to it that would cause race conditions when ran multiple times.